tests/optional_chaining/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`chaining.js 1`] = `
+var street = user.address?.street
+var fooValue = myForm.querySelector('input[name=foo]')?.value
+
+obj?.prop;
+obj?.[expr];
+func?.(...args);
+
+a?.();
```

Comment:
CoffeeScript made it work by not having ternaries at all. (This has the amusing effect that some code, like `a=b?(c,d)+e:d`, is both valid CoffeeScript and valid JS but with totally different semantics in each case.)

Life is much easier when you don't have to worry about old code.